<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_preview_desc</class>
 <widget class="QWidget" name="Form_preview_desc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview description</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="text_desc">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,0,0,0,0,0">
     <item>
      <widget class="QComboBox" name="comboBox_templates"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save_file">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Save to file</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/filmonaizer.qrc">
         <normaloff>:/icons/disk-black.png</normaloff>:/icons/disk-black.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save_buffer">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Save to buffer</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/filmonaizer.qrc">
         <normaloff>:/icons/clipboard-sign.png</normaloff>:/icons/clipboard-sign.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Send to Facebook</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/filmonaizer.qrc">
         <normaloff>:/icons/new/share-balloon.png</normaloff>:/icons/new/share-balloon.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Close window</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/filmonaizer.qrc"/>
 </resources>
 <connections/>
</ui>
